ArduCopter/tuning.cpp

```diff
@@ -7,23 +7,23 @@
 
 // tuning - updates parameters based on the ch6 tuning knob's position
 //  should be called at 3.3hz
-void Copter::tuning() {
-    RC_Channel *rc6 = rc().channel(CH_6);
+void Copter::tuning()
+{
+    const RC_Channel *rc6 = rc().channel(CH_6);
```

Comment:
If we're changing things up, we could take the opportunity to remove this hard-channel-6 dependency by adding a channel-option `AUX_FUNC::radio_tuning` and use `find_channel_for_option` here to get our `RC_Channel*`.

This change would help avoid someone specifying both a function on channel 6 *and* using the channel for tuning.

Reply:
.... if we don't want to go there we could at least add a sanity check here such that if ch6 *does* have a function on it that we don't use it for tuning.

Reply:
Not keen on the scope creep but I'll add in the sanity check, txs